templates/ai/content/ai-basics/_sidebar.html

```diff
@@ -7,7 +7,8 @@
 <li><a href="/ai/content/ai-basics/index.html#why-are-people-excited-about-llms">Why are people excited about LLMs?</a></li>
 <li><a href="/ai/content/ai-basics/index.html#why-are-people-concerned-about-llms">Why are people concerned about LLMs?</a></li>
 <li><a href="/ai/content/ai-basics/index.html#what-exactly-is-an-llm">What exactly is an LLM?</a></li>
-<li><a href="/ai/content/ai-basics/index.html#what-are-the-pros--cons-of-using-an-llm">What are the pros & cons of using an LLM?</a></li>
+
+<li><a href="/ai/content/ai-basics/index.html#what-are-the-pros-cons-of-using-an-llm">What are the pros & cons of using an LLM?</a></li>
```

Comment:
Q: should this be "-pros--cons-` or `-pros-cons-`?
I think currently in staging this is broken and the element/`id` is `<h4 id="what-are-the-pros--cons-of-using-an-llm">What are the pros &amp; cons of using an LLM?</h4>`

So changing this back from a double `--` to single `-` might possibly re-break the sidebar anchor link.